nwbwriter.py

```diff
@@ -32,8 +81,8 @@ def save_NWB(CIDataContainerObj,filename,imaging_plane_name=None,imaging_series_
             indicator: str
             location: str
         """
-        imaging_rate=CIDataContainerObj.SampFreq
-        raw_data_file=CIDataContainerObj.raw_data_file
+        imaging_rate=TraceExtractor.SampFreq
```

Comment:
please use underscore_case for attributes and functions/methods and CamelCase for class names